### PR TITLE
Fixes job covered calculations

### PR DESCRIPTION
**Pull request by ciecierski**
Use has_blockers from blockers.py instead of exception-based approach

### CODE REVIEW

**Review by nathan-weinberg — COMMENTED** [review]
Worked for Enterprise report, will wait to hear from others

**Comment by ciecierski** [comment]
Moving to WIP status. It won't work in situation when there are missing job names in blockers file, for UNSTABLE and FAILED jobs.

File: jeeves/blockers.py [hunk 1]
```diff
@@ -172,9 +172,14 @@ def has_blockers(blockers, job_name):
 	''' returns True if job_name in blockers has any defined blockers
 		returns False otherwise
```
Comment (nathan-weinberg) [inline jeeves/blockers.py:173]:
```suggestion
	''' returns True if job_name exists in blockers and has any defined blockers
		returns False otherwise
```

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
One nit but otherwsie LGTM, @agarcia64 does this resolve your issue?

**Review by agarcia64 — APPROVED** [review]
Just tested on my machine, everything is working as expected and the code looks good to me!

**Review by nathan-weinberg — APPROVED** [review]

**Pull request merged**